src/lib/github.js

```diff
@@ -10,8 +10,8 @@ export const repoInfo = repo =>
 		.then(checkStatus)
 		.then(r => r.json())
 		.catch(() => ({
-			stargazers_count: 9999,
-			watchers_count: 9999
+			stargazers_count: 35780,
+			watchers_count: 405
```

Comment:
I mentioned this in Slack, but our lambda functions are current borked in prod, resulting in this fallback data always shown, which is quite incorrect.